### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by BigDataHa** [protossw512/PRNet_training#2]
I wrote the training code, but it doesn't work. could I ask you some questions

**Comment by tensorboy** [comment]
Hi, @BigDataHa  I'm also working on this project, but I found it's not easy to make the model right as PyTorch didn't have the 'same' padding for conv and deconv operation. I would love to discuss if you want. :)

**Comment by BigDataHa** [comment]
Hi, @tensorboy, I wrote the code with tensorflow, The net is provided by the authors. The model is convergent, but the test results are poor. 17719490563163.com is my email address.

**Comment by BigDataHa** [comment]
Hi, @tensorboy, I wrote the code with tensorflow, The net is provided by the authors. The model is convergent, but the test results are poor. 17719490563@163.com is my email address.

**Comment by Beitadoge** [comment]
@BigDataHa hi i try to implement the train code ,but i didn't convergent,could you share your train code with me ? and we can discuss together~my email is 627153559@qq.com,thanks very much!
